balena/models/device_type.py

```diff
@@ -0,0 +1,126 @@
+from ..base_request import BaseRequest
+from ..settings import Settings
+from .. import exceptions
+from ..utils import is_id
+
+
+class DeviceType(object):
```

Comment:
Idea for a follow-up: Now that we have a dedicated DeviceType model, it might be a chance to deprecate some of the methods of the Device model like:
device.get_display_name
device.get_device_slug
device.get_supported_device_types